docs/db_dev_process/parts.rst

```diff
@@ -1,118 +1,3 @@
-
```

Comment:
Out of interest, why is this file moving?

Reply:
So, there was a main issue in cross-referencing links in Markdown files that were directed to some repo's directory (e.g. https://github.com/SymbiFlow/prjxray/blob/master/README.md#minitests).
To solve this I have split the `parts.rst` into separate sections (minitests and fuzzers) and moved them into their corresponding folder.
After that I have created a symlink to the `minitests` and `fuzzers` folder in the `code` tree so that the `code2docs` `docs2code` mapping would contain also these cross-references used to solve the pending_xref nodes containing the wrong link to the `minitests` or `fuzzers` directory.

Reply:
I'm not sure I understand this? The process to generate the link should be;
  1) Lookup src in `code2docs`
  2) Lookup dst in `code2docs`
  3) Calculate the relative locations of src and dst files - use this as the pending_xref value

Reply:
Partly explained it [here](https://github.com/SymbiFlow/prjxray/pull/1040#discussion_r327719854).
The process is the one that you described, the only issue is that there was no initial mapping in `code2docs` neither in `docs2code` for the minitest and fuzzers directories. To do so there was the need for a fake sym-link that had to be called `index`. 
Therefore I have split the fuzzers and minitests in separate directories.